doc/install_mne_python.rst

```diff
@@ -2,101 +2,112 @@
 
 .. _install_python_and_mne_python:
 
-Installing MNE-Python
----------------------
-
-There are many ways to install a Python interpreter and MNE. Here we show a simple well tested solution.
-
-1. Get Python
-#############
-
-MNE-Python 0.18 only supports Python 3.5+.
-We recommend the `Anaconda distribution <https://www.anaconda.com/distribution/>`_.
-Follow the `installation instructions <http://docs.continuum.io/anaconda/install>`_.
-When you are done, you should see a similar output if you type the following command in a terminal:
+Installing MNE-python
+=====================
+
+.. contents::
+   :local:
+   :depth: 1
+
+Installing Python
+^^^^^^^^^^^^^^^^^
+
+MNE-python runs within python, and depends on several other python packages.
+MNE-python 0.18 only supports Python version 3.5 or higher. We strongly
+recommend the `Anaconda`_ or `Miniconda`_ distributions of python. The main
+difference is that Anaconda comes with more than 250 scientific packages
+pre-bundled, whereas Miniconda starts off with a minimal set of around 30
+packages. Both distributions include the ``conda`` command line tool for
+installing new packages and managing different package sets ("environments")
+for different projects. Follow the installation instructions for `Anaconda
+<anaconda-install>`__ or `Miniconda <miniconda-install>`__; when you are done,
+you should see a similar output if you type the following command in a
+terminal:
```

Comment:
I wouldn't add stuff about `miniconda` here. There are a ton of ways to install Python in general, and MNE works with most of them. I'd rather promote one path here, which is what 95% of users will want. The 5% are advanced enough to learn about other methods (miniconda included) themselves.

If you really want to keep info about it, let's add something to the advanced install/debugging page about how you could use miniconda. Eventually we can mentin that `pip install --user requirements.txt` with Linux generally works, too, if we want, but stuff like that can be separate.